tony-core/src/main/java/com/linkedin/tony/TonyApplicationMaster.java

```diff
@@ -160,6 +160,9 @@
   // Failure conditions
   private boolean jobFailed;
 
+  // Handle different machine frameworks
+  private TonyConfigurationKeys.MLFramework framework;
```

Comment:
Can you import `TonyConfigurationKeys.MLFramework` and just use `MLFramework` here and elsewhere in this file?